Doc/library/time.rst

```diff
@@ -774,6 +774,16 @@ These constants are used as parameters for :func:`clock_getres` and
 
    .. versionadded:: 3.7
 
+.. data:: CLOCK_TAI
+
+   International atomic time, as seconds from the unix epoch.
```

Comment:
It seems erroneous to say "seconds" here, since the constant merely implies a a clock not a unit. It might be helpful to have a sentence about how International Atomic Time—which should be capitalized by the way—is different from UTC. Or at least provide a link to an official page about it if there is one.

Reply:
I will add a link though I worry that the long line will be unwelcome.

Note that computer clock time is not actually UTC (my text in the issue was not fully correct), though it is closely related. This makes it difficult to explain in a few words what this does. But anyone who wants TAI will be pleased to have CLOCK_TAI.

There is also a CLOCK_UTC constant I could expose, but I do not see how it can be used to return anything useful. For proper UTC handling at a leap second it is necessary to express the date as a datetime object, ISO format, or similar; a duration since the epoch is not good enough.